app/views/spaces/edit/common/_edit_rich_text.html.erb

```diff
@@ -7,7 +7,7 @@
     tag.h3 t("space_show.only_for_space", title: @space.title) if SpaceGroup.method_defined? field
   %>
 
-  <%= form.input field, as: :rich_text_area, label: false %>
+  <%= form.input field, as: :rich_text_area, label: false, input_html: { data: { controller: "trix-paste" } } %>
```

Comment:
er det mulig å sette denne til alle :rich_text_area, og ikke måtte spesifisere det hver gang vi bruker rich_text_area?

Reply:
Hmm er ikke helt sikker på hvordan vi skulle gjort det, kan eventuelt være en helper.

Reply:
Gjorde litt research på det nå. 

Jeg har to leads, men ikke noe svar.

# Sannsynlig best: Konfigurere simple_form så den alltid er satt på rich_text_area, og bruke rich_text_area direkte sånn her: 

`form.rich_text_area`

Men vet ikke hvordan konfigurere data-attributten for alle rich_text_areas, selv om jeg regner med at det går an!

# Mulig løsning: Lage et custom input felt "lnu_rich_text_area" og ha det der: 

Har en slags proof-of-concept her, hvor jeg i det minste kan legge FOOBAR foran et rich text area:

![image](https://user-images.githubusercontent.com/14905290/145666980-fd4a7898-c18f-4193-af05-159208238951.png)

Her er det da en ny fil "lnu_rich_text_area_input.rb" som ligger i app/inputs.

Den extender SimpleForm sitt RichTextAreaInput:

```ruby
class LnuRichTextInput < SimpleForm::Inputs::RichTextAreaInput
  def input(wrapper_options = nil)
    merged_input_options = merge_wrapper_options(input_html_options, wrapper_options)

    "FOOBAR #{@builder.rich_text_area(attribute_name, merged_input_options)}"
  end
end
```

som så kan brukes sånn her: 

````
  <%= form.input field, as: :lnu_rich_text %>
```

Men tror det egentlig er best å konfigurere det elementet vi allerede har, heller enn å lage et nytt ett - og da kanskje helst gjennom konfigurasjonsfila til simple_form - om det går an! 

Kunne du spørre noen på huset som kan simple_form godt?

Reply:
Ah tenkte ikke på at vi kan bruke simple forms til dette har oppdatert nå til at alle rich text får paste controller